### PR TITLE
Added callback option to startServer()

### PR DESCRIPTION
**Pull request by jlndk**
Summary
------------
+ Added a callback option to startServer(), so developers can extend
polyServe, if they want. 
+ Solves #31

### CODE REVIEW

**Comment by joao-parana** [comment]
+1

File: src/start_server.js [hunk 1]
```diff
@@ -69,6 +69,10 @@ function startWithPort(options) {
       opn(url, options.browser);
     }
   }
+  
+  if (options.callback) {
+	options.callback(app, server);
```
Comment (ebidel) [inline src/start_server.js]: 2 space indent for consistency

**Comment by justinfagnani** [comment]
I'd much rather return a Promise than accept a callback.

**Comment by justinfagnani** [comment]
I made `startServer` return a Promise.

**Pull request closed without merging**